imports/lib/models/Hunts.ts

```diff
@@ -68,6 +71,7 @@ export const HuntPattern = {
   puzzleHooksDiscordChannel: Match.Optional(SavedDiscordObjectPattern),
   firehoseDiscordChannel: Match.Optional(SavedDiscordObjectPattern),
   memberDiscordRole: Match.Optional(SavedDiscordObjectPattern),
+  invitationCode: Match.Optional(String),
```

Comment:
`HuntPattern` is used by the `createHunt` and `updateHunt` methods to determine what parameters users are allowed to set. I think we want users going through the new methods, rather than setting it directly, so I think we can drop this

Reply:
Ack, removed as part of move to a dedicated model for invitation codes.